### DISCUSSION TITLE
Backport for WOLK

### DISCUSSION THREAD

**Discussion by varjla** [Hekili/hekili#1595] [Q&A] [answered]
Hello all:

After asking Zydras to make a backport of this addon, I was adviced to not to do it as you need to recode the old rotations manually.

Maybe, someone did it yet and I cannot find the correct project? Maybe I need to think to port another addon like Spellflash (which maybe the backport from the version 4.0.3 could be easier)?

Maybe I must use my brain to play, yes, But I prefer to not to do so :D

**Comment by Hekili** [comment, marked as answer]
I'd encourage using your brain regardless of whether I release a Wrath version.

I'm exploring whether to pursue it and have done some initial research.  Having accurate priorities is pretty critical, and I have some doubts about what exists out in the wild.  But I might end up with a basic framework for writing a priority that people could use.  IDK.  Thinking about it.  No promises.
